### PR TITLE
Fix remove style level attr from node

### PR DESCRIPTION
**Pull request by firmusoft**
1. Removed style attributes from node.
2. Implemented Next line style settings on enter key.
3. Number hierarchy managed on delete.

### CODE REVIEW

File: src/ListToggleCommand.js [hunk 1]
```diff
@@ -21,21 +22,29 @@ class ListToggleCommand extends UICommand {
   }
 
   isActive = (state: EditorState): boolean => {
+    let bOK = false;
```
Comment (melgish) [inline src/ListToggleCommand.js:25]: Please provide explanation why this function had to be changed.

**Review by melgish — APPROVED** [review]
Not pleased with all the unnecessary whitespace changes occurring commit after commit.  Need to takes steps to minimize these in future.

**Comment by firmusoft** [comment]
> Not pleased with all the unnecessary whitespace changes occurring commit after commit. Need to takes steps to minimize these in future.

We have run the lint command and format code before each commit, the issue occurs because of resolving merge conflicts.
After committing the merge conflicts, we didn't run the lint command, shall make sure to avoid such issues.

File: src/ListToggleCommand.js (hunk 1 above)
Comment (firmusoft) [reply src/ListToggleCommand.js:25]: To avoid multiple return from the function we modified it.

**Pull request merged**